cmd/amp/platform_monitor.go

```diff
@@ -9,7 +9,7 @@ import (
 var PlatformMonitor = &cobra.Command{
 	Use:   "monitor",
 	Short: "Display AMP platform services",
-	Long:  `Display AMP platform services information and states.`,
+	Long:  `Display command displays information about AMP platform services and states.`,
```

Comment:
`Display command displays information about AMP platform services and states`
I think this should this be monitor ?